app/services/users.js

```diff
@@ -23,3 +25,17 @@ exports.createUser = async ({ firstName, lastName, email, password }) => {
     throw errors.conflictError(EMAIL_ALREADY_ERROR);
   }
 };
+
+exports.signIn = async (email, password) => {
```

Comment:
we have a security breach in this method because we respond to an error if the email is wrong and anything when is ok, so the attacker knows that he has already found the email and only needs the password. I recommend  to you use a unique message for both cases, something like "email and/or password incorrect"

Reply:
solved. Thanks by feedback.